ManiVault/src/PluginFactory.h

```diff
@@ -117,6 +117,7 @@ class CORE_EXPORT PluginFactory : public QObject
      * Get the read me markdown file URL
      * @return URL of the read me markdown file
      */
+    [[deprecated("This function is under development and has no effect (yet).")]]
     virtual QUrl getReadmeMarkdownUrl() const;
```

Comment:
These functions are actually used now, right? Do they still need the `deprecated` attribute? They cause a bunch of warning during build

Reply:
They have largely been superseded by the tutorials, so I am not sure yet what role they will play in the future. I could remove the deprecation warning or remove these functions entirely, what do you think?

Reply:
I think the function to just show the Readme of a plugin without an extra tutorial is still very nice. It can server different purposes as well, like mentioning authors and references in the readme vs detailed usage instructions in the tutorial.
I'd be in favor of keeping it.